.github/workflows/itest-deploy.yml

```diff
@@ -13,6 +13,62 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Initialize Comment
+        id: create_comment
+        run: |
+          COMMENT_BODY="{ \"body\": \"| Step | Status |\\n|------|--------|\\n| Initialize Workflow |\" }"
+          COMMENT_RESPONSE=$(curl -s -X POST \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github+json" \
+            -d "$COMMENT_BODY" \
+            https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments)
```

Comment:
This seems to comment on an issue, but we need to add a comment to the pull request.

Reply:
After checking the API docs, seems I was wrong, since this is not a review comment.